pkg/shell/shell-modals.jsx

```diff
@@ -133,11 +133,27 @@ export const LangModal = () => {
                     <Divider />
                     <MenuContent>
                         <MenuList>
-                            {Object.keys(manifest.locales || { })
-                                    .filter(key => !searchInput || manifest.locales[key].toLowerCase().includes(searchInput.toString().toLowerCase()))
-                                    .map(key => {
-                                        return <MenuItem itemId={key} key={key} data-value={key}>{manifest.locales[key]}</MenuItem>;
-                                    })}
+                            {
+                                (() => {
+                                    const filteredLocales = Object.keys(manifest.locales || {})
+                                            .filter(key => !searchInput || manifest.locales[key].toLowerCase().includes(searchInput.toString().toLowerCase()));
+
+                                    if (filteredLocales.length === 0) {
+                                        return (
+                                            <MenuItem>
```

Comment:
This needs to have isDisabled property.

Reply:
You can indeed change the text, and color will be auto-adjusted when you use the isDisabled. And I think it's already correctly aligned.

Reply:
Yes I have tried it.
Actually it is being planned to fix the size of inside language box(one with grey borders), so for example if we are making it of height 150px, "No language found" text will be shown on the top left side of the box.
![current](https://user-images.githubusercontent.com/31029314/234326638-6eccf5c4-2182-457b-a64a-879591ed2c58.png)

Instead we want it to be aligned to center, vertically and horizontally.
![expected](https://user-images.githubusercontent.com/31029314/234326693-799604a9-0ef4-46f8-aaf6-49912a7fb3f7.png)

So, I am planning to remove the MenuItem and use a div directly with some inline css applied to it.
Other option is to use EmptyState component.

Reply:
No please don't that. The way you do it right now is correct and according to the PF4 docs. https://www.patternfly.org/v4/components/select/#grouped-typeahead

Reply:
In order to fix the jumping issue, better give the whole modal body a higher height, rather than giving the empty state menuitem a bigger height. Does this make sense @shubhit7 ?

Reply:
If we give the whole modal a higher height, the jumping issue will still be there as jumping is happening due the language box which is inside. 
So we need to give this language box(one with grey borders) a higher fixed height, once this is done we have to give the height as 100% to its children as this box acts as container of menuitems.
I am not giving the empty state menuitem a bigger height, I am making them 100% which they will relatively adjust according to their container size.

I am not resolving the jumping issue in this PR, here I am just suggesting to go ahead with EmptyState component inside MenuItem, that is pretty much it.
So that I can change the heights of these in a different PR.

Reply:
@KKoukiou There is only a slight change.
Instead of menuitem we can also directly use:
`return (
<EmptyState>
    No match
</EmptyState>
)
`

Reply:
The Menu element creates a `<ul>`  HTML tag. MenuItem is `<li>`
The only valid element as a child of ul is the li. See https://stackoverflow.com/questions/11755628/can-i-use-div-as-a-direct-child-of-ul

By skipping the usage of MenuItem you essentially create invalid CSS. If you want to center align the text use the alignment classes from https://www.patternfly.org/v4/utilities/alignment#basic.

If you want to specify the height of the menu do it with custom CSS.